docs/environment-setup.md

```diff
@@ -137,67 +137,76 @@ _NOTE: This will not work on WSL (Windows Subsystem for Linux). Use the approach
 
 ## After installing the prerequisites:
 
-In the root directory, run `cp env.example .env` on Linux/Mac or `copy env.example .env`.
-
-_Note: The `env.example` file has examples commented on top of each variable_
-
-### Now you can start Redis and Elasticsearch without errors using one of the following methods:
-
-_Note: Make sure only one of the below options are used when starting Telescope_
-
-#### Option 1: Using Docker
+### Start Docker
 
 ```
 sudo systemctl start docker
-docker-compose up --build redis elasticsearch
 ```
 
 _Note: You may need to add your user to the docker group in Linux to use `docker-compose` without `sudo`. To do this, try `groups $USER` in a terminal and check if docker is in the list of groups. If not, add it with `usermod -aG docker $USER` and reboot._
 
 **Important:** Docker builds Telescope's dependencies at launch and keeps them on disk. In some cases, Docker might try to reuse already-built dependencies or cached data, causing misleading results when testing Telescope. To avoid this, it is recommended to use the command `docker system prune -af --volumes` to remove all already-built Telescope dependencies and ensure fresh deployments.
 More information about docker: [images vs containers](https://www.baeldung.com/docker-images-vs-containers) and [volumes](https://docs.docker.com/storage/volumes/).
 
-#### Option 2: Natively installed:
+### Start telescope
+
+There are different ways to run the application. By default, `.env` will be read; however, if there is no `.env` file in the root, [env.development](../config/env.development) will be used.
+
+There are also [env.production](../config/env.production) and [env.staging](../config/env.staging) to choose based on developer's need. For example, if you want to use staging, you can do `cp config/env.staging .env` on Linux/macOS, or `copy config/env.staging .env` on windows.
+
+Here are instructions for different scenarios:
+_Note: Make sure you're running these commands in the root of telescope project. If any of the commands below are failing, use the command `pwd` to find your current directory and navigate back to project root (e.g., `cd <the path of directory you place telescope project>/telescope`)_
+
+#### Want to run back-end services locally, and to have front-end using the services
+
+This is the default setting, you do not need to copy or modify any `env` file.
 
-#### Redis:
+```
```

Comment:
Should we add `bash` after the opening '```' to support highlighting?

Reply:
yes, thank you!